tests/lib/screenshot-testing/support/chai-extras.js

```diff
@@ -42,6 +42,13 @@ function getPageLogsString(pageLogs, indent) {
     return result;
 }
 
+function logErrorMessage(message, indent) {
+    var indent = "     ";
+    console.log("\n\n" + indent + 'This message usually belongs to the next test');
+    message = message.replace(/\n/g, "\n" + indent);
+    console.log(indent + message);
+}
```

Comment:
Instead of using console.log which will display the message before the test's summary, maybe you could use a mocha hook to display the information. Eg, something like:

```
app.runner.on('pass', function() {
    console.log('print all the messages');
});
```

`app` is a global object. `app.runner` is created in app.js in `Application.prototype.doRunTests`.